src/app/(post-login)/submission/program/[shortName]/clinical-submission/components/Header.tsx

```diff
@@ -91,6 +115,43 @@ const Header: FC<HeaderProps> = ({
 		},
 	});
 
+	const handleSubmissionApproval = async () => {
+		const didUserConfirm = await getUserConfirmation({
+			title: 'Approve Submission?',
+			children: 'Are you sure you want to approve this clinical submission?',
+			actionButtonText: 'Yes, Approve',
+			actionButtonId: 'modal-confirm-approve',
+			buttonSize: 'sm',
+		});
+
+		if (didUserConfirm) {
+			setGlobalLoading(true);
+			await sleep();
+			try {
+				await approveClinicalSubmission();
+
+				updateClinicalSubmissionQuery((previous: any) => ({
+					...previous,
+					clinicalSubmissions:
+						placeholderClinicalSubmissionQueryData(programShortName).clinicalSubmissions,
```

Comment:
Stating the obvious but placeholder will only be temporary, which ticket(s) is this PR + follow up connected to?